lib/matplotlib/rcsetup.py

```diff
@@ -93,6 +94,21 @@ def __call__(self, s):
         raise ValueError(msg)
 
 
+def _single_string_color_list(s, scalar_validator):
+    try:
+        colors = mpl.color_sequences[s]
+    except KeyError as ke:
```

Comment:
Doesn't this raise a `TypeError` if `s` is a list?

Reply:
This function is only called if `s` is a string
https://github.com/rcomer/matplotlib/blob/846e8259e2cc7de0c982faedfb694c89a47e1ac3/lib/matplotlib/rcsetup.py#L116-L123